packages/core/src/Machine.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { StateNode } from './StateNode';
 
 export function Machine<
-  TContext = any,
+  TContext = unknown,
```

Comment:
I believe the change is good, but it's also a breaking change, are we OK with that? I am 😉

Reply:
Hmm good catch, maybe not right now. Let's save this for V5